src/theme/Footer/index.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React from 'react';
+import styles from './styles.module.css';
+
+const content = {
+  sponsorsContent: (
+    <>
+      If you like our work, you can <b>sponsor & support</b> the community!<br></br>
+      All the money will be used to continue our mission of <b>providing simplified cloud native education to all</b> and make this community reach more number of people.
+    </>
+  )
+}
+
+const logo = {
+  hashnode: { 
+    Svg: require('@site/static/img/Vector.svg').default
+  },
+  coffee : {
+    Svg: require('@site/static/img/coffee.svg').default
+  },
+  mail: {
+    Svg: require('@site/static/img/mail.svg').default
+  },
+  instagram: {
+    Svg: require('@site/static/img/instagram.svg').default
+  },
+  twitter: {
+    Svg: require('@site/static/img/twitter.svg').default
+  },
+  github: {
+    Svg: require('@site/static/img/github.svg').default
+  },
+  youtube: {
+    Svg: require('@site/static/img/youtube.svg').default
+  },
+  linkedin: {
+    Svg: require('@site/static/img/linkedIn.svg').default
+  }
+}
+
+function Footer() {
+  return (
+    <section className={styles.footer}>
+      <div>
+        <div className={styles.title}>help us do more</div>
```

Comment:
Heading should start with a capital letter.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix